### PR TITLE
RHIDP-2061: Update image name and other doc improvements

### PR DESCRIPTION
**Pull request by abrennan89**
**Versions:**
1.1+

**Jira link:**
- https://issues.redhat.com/browse/RHIDP-2061

**Preview link:**
https://redhat-developer.github.io/red-hat-developers-documentation-rhdh/pr-244/admin-rhdh/#proc-install-rhdh-ocp-helm_admin-rhdh

**Reviews:**
- [x] SME: @nickboldt 
- QE: N/A, mostly language changes and one small change to the image that was confirmed by SMEs 
- [X] Peer review: @linfraze

### CODE REVIEW

File: modules/installation/proc-install-rhdh-airgapped-environment-ocp-helm.adoc [hunk 1]
```diff
@@ -18,93 +23,90 @@ To install {product-short} in an air-gapped environment, you must have access to
...
---
 
-. Pull {product-short} and PostgreSQL images from https://catalog.redhat.com/software/containers/search[{company-name} Image registry] to your workstation:
+. Pull {product-short} and PostgreSQL images from https://catalog.redhat.com/software/containers/search[{company-name} Image registry] to your workstation, by running the following commands:
 +
---
-[source]
+[source,terminal]
+----
+podman pull registry.redhat.io/rhdh/rhdh-hub-rhel9:{product-version}
```
Comment (nickboldt) [inline modules/installation/proc-install-rhdh-airgapped-environment-ocp-helm.adoc:68]: See https://github.com/redhat-developer/red-hat-developers-documentation-rhdh/blob/main/artifacts/attributes.adoc#L15

```suggestion
podman pull registry.redhat.io/rhdh/rhdh-hub-rhel9:{product-chart-version}
```

**Review by nickboldt — APPROVED** [review]

**Comment by nickboldt** [comment]
Since this is mostly just language changes, +1 from me. One small change to the container image reference (use the chart version, not the RHDH version)

**Comment by nickboldt** [comment]
Note that before GA these should be changed to 1.2.0, or you'll document in the 1.2 docs how to install the 1.1 release. 

https://github.com/redhat-developer/red-hat-developers-documentation-rhdh/blob/main/artifacts/attributes.adoc#L14-L15

**Review by linfraze — APPROVED** [review]
/lgtm

**Pull request merged**